pkgs/tools/virtualization/tart/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "tart";
+  version = "0.19.0";
+
+  src = fetchurl {
+    url = "https://github.com/cirruslabs/${pname}/releases/download/${version}/${pname}.tar.gz";
+    sha256 = "sha256-q7zihIaVyN42xif67PGDuSB+vq1dlM2wGVn+oaa1itE=";
+  };
+
+  dontBuild = true;
+
+  unpackPhase = ''
+    mkdir source/
+    tar xvf $src -C source/
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    install -Dm755 source/tart -t $out/bin
```

```suggestion
    install -Dm755 tart -t $out/bin
```

`install -D` will create the target directory.